packages/schema-blocks/src/core/validation/BlockValidationResult.ts

```diff
@@ -1,6 +1,13 @@
 import { BlockValidation } from ".";
 import { BlockInstance } from "@wordpress/blocks";
 
+export enum BlockType {
```

Comment:
Blocks hebben al een type; "yoast/job-posting" bijvoorbeeld. Kunnen we een meer beschrijvende naam verzinnen? 
```suggestion
export enum BlockConstraint {
```

Reply:
`"yoast/job-posting"` is technisch gezien de `name` 😉 . Maar ik zie je punt - wat vind je van `BlockPresence`?

Reply:
Conclusie: hernoemd naar `BlockPresence`.